src/layouts/CourseLayout.tsx

```diff
@@ -10,7 +10,9 @@ import { getTableOfContent } from '../utils/getTableOfContent';
 import { useTableOfContents } from '../hooks/useTableOfContent';
 import { Tag } from '../components/Tag';
 import { NavigationItem } from '../constants/navigations';
-import { useRouter } from 'next/router';
+import { Router, useRouter } from 'next/router';
+import MobileCourseNav from 'src/components/MobileCourseNav';
```

Comment:
- Use named import like `import { MobileCourseNave } from '../components/MobileCourseNav';`
- Also use relative path during import like `../components/MobileCourseNav` instead of `src/components/MobileCourseNav`